### PR TITLE
[Fix] Typos in .gitignore & error in customize_runtime docs

### PR DESCRIPTION
**Pull request by Wuziyi616**
The ignored data folder should be `data/xxx` instead of `/data/xxx`.
MMDet no longer has `mmdet/core/optimizer`. Instead, they use optimizer from `mmcv`. I modify this part of docs accordingly.

### CODE REVIEW

File: docs/tutorials/customize_runtime.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ You need to create a new directory named `mmdet/core/optimizer`.
 And then implement the new optimizer in a file, e.g., in `mmdet/core/optimizer/my_optimizer.py`:
```
Comment (Wuziyi616) [inline docs/tutorials/customize_runtime.md:24]: Where do you think is better to add the file? If we add at `mmdet`, we assume the user installed `mmdet ` via git clone. This is the same if we add at `mmcv`. I would prefer to add this file `mmdet3d/core/optimizer/` which is the easiest to implement.

**Comment by Wuziyi616** [comment]
This issue may exist in all downstream repos because `optimizer` is now implemented in `mmcv`. At least the customize_runtime docs of MMDet and MMSeg are both wrong.

**Comment by Wuziyi616** [comment]
The CI error is because MMDet has updated to use MMCV's MODEL registry. This check can be passed after merging #495.

File: docs/tutorials/customize_runtime.md (hunk 1 above)
Comment (ZwwWayne) [reply docs/tutorials/customize_runtime.md:24]: should add to mmdet3d since mmdet is taken as a lib by default

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**